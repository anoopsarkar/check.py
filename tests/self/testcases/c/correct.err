running build nl
running nl 1
running nl 2
running nl 3
running nl 4
