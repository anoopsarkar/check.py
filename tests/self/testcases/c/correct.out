build nl : passed
nl 1 : passed
nl 2 : passed
nl 3 : passed
nl 4 : passed
nl : 4 / 4
build : 1 / 1
total : 5 / 5
